optimize_sidebars.js

```diff
@@ -1,182 +1,2 @@
-          type: "link",
-          label: "Introduction",
-          href: "/docs/components/operate/index",
-        },
-        {
-          "User guide": [
-            {
-              type: "link",
-              label: "Getting familiar with Operate",
-              href: "/docs/components/operate/userguide/basic-operate-navigation",
-            },
-            {
-              type: "link",
-              label: "Variables and incidents",
-              href: "/docs/components/operate/userguide/resolve-incidents-update-variables",
-            },
-          ],
-        },
-      ],
-
-      // POC note: this is the only section of links that _doesn't_ cross over to the main docs instance.
-      //   Anything here can use the shortcut format.
-      Optimize: [
-        "what-is-optimize",
-        {
-          "User guide": [
-            "userguide/collections-dashboards-reports",
-            "userguide/data-sources",
-            {
-              "Process analysis": [
-                "userguide/process-analysis/overview",
-                "userguide/process-analysis/outlier-analysis",
-              ],
-            },
-          ],
-        },
-      ],
-
-      Tasklist: [
-        {
-          type: "link",
-          label: "Introduction",
-          href: "/docs/components/tasklist/introduction",
-        },
-        {
-          "User guide": [
-            {
-              type: "link",
-              label: "Overview and example use case",
-              href: "/docs/components/tasklist/userguide/overview",
-            },
-            {
-              type: "link",
-              label: "Updating Tasklist",
-              href: "/docs/components/tasklist/userguide/updating-tasklist-cloud",
-            },
-          ],
-        },
-      ],
-      "Best Practices": [
-        {
-          type: "link",
-          label: "Overview",
-          href: "/docs/components/best-practices/overview",
-        },
-        {
-          "Project Management": [
-            {
-              type: "link",
-              label: "Following the Customer Success Path",
-              href: "/docs/components/best-practices/management/following-the-customer-success-path",
-            },
-            {
-              type: "link",
-              label: "Doing a proper POC",
-              href: "/docs/components/best-practices/management/doing-a-proper-poc",
-            },
-          ],
-          Architecture: [
-            {
-              type: "link",
-              label: "Deciding about your stack",
-              href: "/docs/components/best-practices/architecture/deciding-about-your-stack",
-            },
-            {
-              type: "link",
-              label: "Sizing your environment",
-              href: "/docs/components/best-practices/architecture/sizing-your-environment",
-            },
-          ],
-        },
-      ],
-    },
-  ],
-};
+const sidebars = require("./sidebars.js");
+module.exports = sidebars;
```

Comment:
This is the main goal of this PR -- remove the duplication of the entire structure across the two sidebars files.